gesturecontrol.py

```diff
@@ -168,26 +173,38 @@ def stream(self, saveData = False):
 		if writecsv:
 			self.createCSV(vectorDataSet)
 
-	def playback(self):
-		print("[INFO] Playback random values")
+	def playback(self, fileName):
+		print("[INFO] Playback data from csv sheet")
 		self.running = True
 
-		while self.running:
-			time.sleep(0.5)
-			key = cv2.waitKey(1) & 0xFF
-
-			# if the `q` key was pressed, break from the loop
-			if key == ord("q"):
-				# Needs work
-				self.running = False
-				break
+		dataList = self.getCSVData(fileName)
```

Comment:
May be wise to check that the filename is valid and error handling.

It's nit picky but no harm done to handle possible user errors